poc/map/Jack.tbasic

```diff
@@ -418,13 +437,35 @@ Console_ShowWindow(%Console_SW_SHOWMAXIMIZED)
         
     End Select
     
+
     '[todo] <Something to do here>
     ' Here sorting me.worldList(i).arms(j).width wide to narrow ; how array sort ?
-    
+   #Region " Sort arms by width descending"
+    Dim sorted as Boolean
+    DIm tempArm as tArm
+      '[breakpoint] <Any note here. Breakpoint will be set to line following this statement>
+    Do
+      sorted = %TRUE
+      for j = 2 to me.worldList(i).ambiance.riverArmsCount
+        if me.worldList(i).arms(j).width > me.worldList(i).arms(j-1).width Then
+        	sorted = %False
+        	tempArm = me.worldList(i).arms(j).width
+        	me.worldList(i).arms(j-1).width = me.worldList(i).arms(j).width
+        	me.worldList(i).arms(j).width = tempArm
+        End If
+      Next
+    loop until sorted = %TRUE
+   #EndRegion 
+logger.write(%LT_ToConsole, "", "", %TAG_NOTAG)
+logger.wInfo("Setting river properties...")
+logger.write(%LT_ToConsole, "", "", %TAG_NOTAG)
```

Comment:
Hi Petr, 

this branch is not finished, but : 
can you guess why commenting these 3 lines leads to *not* working code ?

Reply:
Can you describe the "not working" state in more detail, if possible?

I could not reproduce it on my PC, but fact is I have slighlty modded "in development" version.

Reply:
First I didn't wanted the wInfo there in the for/next as it is repeated.
If you remove/comment out the 3 lines, then the script is not going further ( the log screen is stuck at "Setting river properties"). Removing the log lines shouldn't change script behaviour.

Reply:
I agree 100%, that the operations you mention should be perfectly safe and without side effect. I cannot replicate it even when commenting out currently, but I do not have vanilla 1.11.6.0. I think Eros maybe fixed it in upcoming version.

Reply:
Well, just tried it here on leaked v1.11.7.0. Same issue. Even replacing by other instructions is blocking. Maybe it's logger related ? I don't know but will think about it.

Reply:
if I replace lines 459 to 461 by `msgbox 0,str$(i) & " - "  & str$(j)` then I get two msgbox, then nothing.
messages are "1 - 2" then "2 - 2"
but !
_j_ should not be defined ! how is this possible ? I didn't find.
I should have got an error from thinBasic. Do you agree ?

Reply:
I agree. I am just nervous why I cannot replicate the behaviour. I will try to install older version.

Reply:
thanks for that :)